### ISSUE TITLE
Copy number doesn't display

### ISSUE THREAD

**Issue by patrickzurek** [CARLI/vufind#21]
JIRA issue created by: Jessica Gibson (gibsonjc)
Originally opened: 2016-01-13 12:35 PM

Traditionally, we have used the copy number from the MFHD 852$t for our OPAC displays; however, given that out-of-the-box VuFind2 isn't displaying much, if any, information from the MFHD as far as I can tell, I can understand why it isn't doing that!

But VuFind2 does display a lot of things from the Voyager Item record, and it has the word "Copy" hardcoded somewhere in its display, it just doesn't seem to be pulling the copy information from the Item record, or it is getting the information but not showing it.

(I don't know if this is a specific problem to solve or if it will be solved another way by ultimately making more information from the MFHDs display instead, but I thought it was unusual and worth noting.)

**Comment by patrickzurek** [comment]
JIRA User: Chris Delis (cedelis)
JIRA Timestamp: 2016-02-22 10:27 AM

Can you provide example bib? Thanks.

**Comment by patrickzurek** [comment]
JIRA User: Jessica Gibson (gibsonjc)
JIRA Timestamp: 2016-03-02 10:18 AM

Copy numbers as found in Voyager Item records are displaying now as:

Copy c. <number>

The word "Copy" and the "c." are redundant and only one is necessary.  Would it be possible to remove the "c." from displaying?  Current production catalogs have the word "Copy:"

Example can be seen in any record that has an Item Record, but here is an example where I added an additional copy 63:
https://vufind2-test.carli.illinois.edu/vf-kcc/Record/(KCCdb)58777

**Comment by patrickzurek** [comment]
JIRA User: Jessica Gibson (gibsonjc)
JIRA Timestamp: 2016-03-24 09:56 AM

When there isn't an item to get a number from, the word Copy still appears by itself, which is a bit untidy.

**Comment by patrickzurek** [comment]
JIRA User: Chris Delis (cedelis)
JIRA Timestamp: 2016-04-11 05:51 PM

I discovered where the c. comes from:

In VXWS, the following config file, e.g., MILdb/tomcat/vxws/context/vxws/ui/eng/vxws.properties:

holdings.item.copynumber.label=c.

If I set it to blank, i.e.,

holdings.item.copynumber.label=

the label will disappear.

I plan to do this soon, but not at the moment, as I'm working a different issue. Just wanted to point it out.
